usaspending_api/recipient/models.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+
+
+class StateData(models.Model):
+    """
+    Model representing State Data/Year
+    """
+    id = models.TextField(primary_key=True)
+    fips = models.TextField(db_index=True)
+    code = models.TextField()
+    name = models.TextField()
+    type = models.TextField()
+    year = models.IntegerField(db_index=True)
+    population = models.BigIntegerField(null=True, blank=True)
+    pop_source = models.TextField(null=True, blank=True)
+    median_household_income = models.DecimalField(null=True, blank=True, decimal_places=2, max_digits=21)
+    mhi_source = models.TextField(null=True, blank=True)  # median household income source
+
+    class Meta:
+        db_table = 'state_data'
+
+    def save(self, *args, **kwargs):
```

Comment:
A word of caution, if ever using `StateData.objects.create(...)`, the `.save()` does not get called, so the subsequent pre-save happening here wouldn't get called.

Reply:
Is it possible/advisable to override it to include the `save()`?

Reply:
I started looking at a way to do this, but instead moved to ensuring the data transformation was happening at the source of the ingestion instead, since outside of the ETL script, we won't be modifying the data (or shouldn't be).

There's room for more investigation to also override the `create` since I was unable to complete it.